R/pinp.R

```diff
@@ -35,21 +35,32 @@
 #'   suffix can be omitted; defaut is no bibliography.}
 #'   \item{\code{doi}}{(Optional but recommended) A free-format entry
 #'   suitable for a doi or url referencing the document or its
-#'   underlying code.}  \item{\code{fontsize}}{(Optional) Document
+#'   underlying code.}  
+#'   \item{\code{fontsize}}{(Optional) Document
 #'   fontsize, default is 9pt.}
 #'   \item{\code{footer_contents}}{(Optional) A free-format entry for
 #'   text placed in the footer, useful to associate with a package or
 #'   volume, default is \sQuote{Package Vignette}.}
+#'   \item{\code{date_subtitle}}{(Optional) An _optional_ free-form text string.
```

Comment:
This irked me a little when I first glanced at his earlier.  First off, it repeats 'optional' twice within three words.  Second, we are old school here and do NOT use markdown.

So I just fixed this and will commit back into your repo and the PR in a moment.